frontend/lib/onboarding/onboarding-step-3.tsx

```diff
@@ -113,16 +123,20 @@ export const createLeaseModals = (
     leaseType: "MARKET_RATE",
     component: () => (
       <LeaseInfoModal
-        title="Market Rate lease"
+        title={li18n._(t`Market Rate lease`)}
         isWarning
         toNextStep={routes.step4}
       >
         <p>
-          <strong className="has-text-danger">Warning:</strong> Taking action
-          against your landlord could provoke retaliation and/or an eviction
-          notice once the Eviction Moratorium is lifted.{" "}
+          <Trans>
+            <strong className="has-text-danger">Warning:</strong> Taking action
+            against your landlord could provoke retaliation and/or an eviction
+            notice once the Eviction Moratorium is lifted.{" "}
```

Comment:
I wonder if this final whitespace should actually be moved outside of the `<Trans>` tag?  My main concern is that the translator might not notice that the sentence they're translating has a single space after the period, and the result would be that there's a missing space in the final translated text which could look weird.

Reply:
true